testing/moosefs/APKBUILD

```diff
@@ -0,0 +1,148 @@
+# Contributor: Michael Pirogov <vbnet.ru@gmail.com>
+# Maintainer: Michael Pirogov <vbnet.ru@gmail.com>
+pkgname="moosefs"
+pkgver="3.0.105"
+pkgrel=0
+pkgdesc="Open Source, Petabyte, Fault-Tolerant, Highly Performing, Scalable Network Distributed File System"
+url="https://moosefs.com"
+arch="all"
+license="GPL-2.0-only"
+makedepends="autoconf automake zlib-dev libpcap-dev libtool fuse-dev python3-dev"
+pkgusers="mfs"
+pkggroups="mfs"
+subpackages="$pkgname-doc $pkgname-client $pkgname-master $pkgname-chunkserver $pkgname-metalogger $pkgname-cgi::noarch $pkgname-cgiserv::noarch"
```

Comment:
Please break this line after 80 lines

Also, I guess this should come with a $pkgname-openrc subpackage.

Reply:
>Please break this line after 80 lines

done

>Also, I guess this should come with a $pkgname-openrc subpackage.

inits belongs to subpackages and I dunno how to add them. I appreciate if you could show how achieve that.

Reply:
Hm, not sure how we handle initd files of subpackages to be honest :thinking: 

Do you know more, @maxice8 ?

Reply:
How we handle them ? poorly.

1. Add an $subpackage-openrc:_subpkg_openrc
2. write out the default_openrc() from abuild but with proper changes for that specific subpackage

Reply:
Nailed it. Thank you :+1: